packages/dd-trace/test/appsec/sdk/set_user.spec.js

```diff
@@ -0,0 +1,141 @@
+'use strict'
+
+const proxyquire = require('proxyquire')
+const agent = require('../../plugins/agent')
+const tracer = require('../../../../../index')
+const appsec = require('../../../src/appsec')
+const Config = require('../../../src/config')
+const getPort = require('get-port')
+const axios = require('axios')
+
+describe('setUser', () => {
+  describe('Check internal callings', () => {
+    const tracer = {}
+    let sdk, mockSetTag, mockRootSpan, getRootSpan
+
+    beforeEach(() => {
+      mockSetTag = sinon.stub()
+      mockRootSpan = {
+        context: () => {
+          return { _tags: { 'usr.id': 'mockUser' } }
+        },
+        setTag: mockSetTag }
+      getRootSpan = sinon.stub().returns(mockRootSpan)
+
+      const { setUser } = proxyquire('../../../src/appsec/sdk/set_user', {
+        './utils': { getRootSpan }
+      })
+
+      const AppsecSdk = proxyquire('../../../src/appsec/sdk', {
+        './set_user': { setUser }
+      })
+
+      sdk = new AppsecSdk(tracer)
```

Comment:
you don't need to make a sdk instance here i think, the sdk index is already tested in `sdk/index.spec.js`, just use setUser() directly